web/src/input_pill.ts

```diff
@@ -177,6 +179,10 @@ export function create<T>(opts: InputPillCreateOptions<T>): InputPillContainer<T
                 opts.has_status = has_status;
             }
 
+            if (store.pill_config?.do_not_show_user_group_size === true) {
+                opts.display_value = item.group_name;
+            }
```

Comment:
It might be nicer to invert the meaning here -- have a `show_user_group_size` parameter that we pass from the other code paths.